internal/gengapic/gengapic.go

```diff
@@ -532,3 +555,26 @@ func snakeToCamel(s string) string {
 	}
 	return sb.String()
 }
+
+func parseRequestHeaders(m *descriptor.MethodDescriptorProto) [][]string {
+	eHTTP, err := proto.GetExtension(m.GetOptions(), annotations.E_Http)
+	if err != nil {
```

Comment:
Now that I think of it, I think we should explicitly check for `ErrMissingExtension`.

Reply:
Just curious, why do you think it’s important to make the distinction? If it’s not there, we just move on. I guess there could be some errors from parsing the bytes but I’m not sure how to reproduce that.

Reply:
That's a good question.I agree it'd be hard to repro this. Errors from parsing the bytes seem the most plausible. I'm can think of something like this:
- We accidentally change the annotation in a backward incompatible way
- Now if we recompile the generator, the descriptor using the old annotation is invalid.
- With the current code, we assume the annotation is missing and move on.

Situations like this is hopefully rare. I find checking for specific errors a good practice so that we can at least complain about it. Up to you though :)

Reply:
That is reasonable to me! I'll add it. Thanks

Reply:
Done